### DISCUSSION TITLE
Choose GitHub repositories to show on user profile

### DISCUSSION THREAD

**Discussion by kkrypt0nn** [forem/forem#19615] [Feature Requests]
Currently the "GitHub Repositories" section on a user's profile seem to showcase the three repositories with the most stars of the linked GitHub account.

I'd be awesome if we could choose which three - or less - repositories we want to showcase on our profile.

**Comment by j471n** [comment]
Hi mate, This functionality is already there you can go to https://dev.to/settings/extensions and then scroll down to GitHub section and then you will be able to select you favorite repo to pin there as shown in the following screenshot


![image](https://github.com/forem/forem/assets/55713505/72f6cab2-308a-4755-a43d-b91539a3c57d)

**Reply by kkrypt0nn** [reply]
Oh I haven't seen that it's hidden there ^-^ Thank you so much :)

Would be awesome to have it on https://dev.to/settings as well as that's where almost everything regarding the profile can be changed - or at least a link to the other page

**Comment by jline9507** [comment]
Changing a repository's visibility
On GitHub.com, navigate to the main page of the repository.
Under your repository name, click Settings. ...
In the "Danger Zone" section[,](https://www.crimsoncoward.com/) to the right of to "Change repository visibility", click Change visibility.
Select a visibility.

**Reply by kkrypt0nn** [reply]
That has nothing to do with what was mentioned above.

PS: The advertising of that weird website in the middle is a failure.
